modules/security-container-content-inside.adoc

```diff
@@ -30,5 +30,5 @@ scanning features, used directly from {op-system-base} or added to {product-titl
 can alert you when
 an image you are using has vulnerabilities. OpenSCAP image scanning is
 available in {op-system-base} and the
-link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html-single/manage_red_hat_quay/index#container-security-operator-setup[Container Security Operator] can be added
+link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html-single/manage_red_hat_quay/index#container-security-operator-setup[{rhq-cso}] can be added
```

Comment:
@stevsmit it doesn't look like this 4.6 cherry pick included the new attribute in the common-attributes.adoc file, so it's not being replaced properly: https://docs.openshift.com/container-platform/4.6/security/container_security/security-container-content.html#security-container-content-inside_security-container-content

Can you submit a fix to 4.6 to add the `rhq-cso` attribute?